src/Items/ItemBow.h

```diff
@@ -69,17 +69,12 @@ class cItemBowHandler :
 		}
 
 		// Create the arrow entity:
-		cArrowEntity * Arrow = new cArrowEntity(*a_Player, Force * 2);
-		if (Arrow == nullptr)
+		auto ArrowPtr = cpp14::make_unique<cArrowEntity>(*a_Player, Force * 2);
+		auto Arrow = ArrowPtr.get();
+		if (!Arrow->Initialize(std::move(ArrowPtr), *a_Player->GetWorld()))
```

Comment:
I think that's only half of the relevant part. The other half is determining the function to call in the first place. Because the function is virtual, it needs to be decided in runtime. The compiler has to do these things (not necessarily in this order):
 - Dereference the `Arrow` pointer and retrieve its virtual function table, from there determine the address of the function to call
 - Evaluate and push all parameters onto the stack (including the hidden `this` parameter)
 - Jump to the function's address (simplified-speaking)

So the problems this code has are:
 1. Does dereferencing `Arrow` happen before evaluating the `std::move`?
 2. Does pushing the `this` happen before the `std::move`?

If I remember correctly, at least (2) is explicitly undefined, so the compiler is free to choose.

For now, fix the code by adding a temporary variable holding the naked pointer; we'll fix it later by refactoring the `Initialize` function away.